Movies_app/Movies_app/Coordinator/BaseCoordinator.swift

```diff
@@ -0,0 +1,36 @@
+// BaseCoordinator.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Базовый координатор
+class BaseCoordinator {
+    // MARK: - Public properties
+
+    var childCoordinators: [BaseCoordinator] = []
```

Comment:
```suggestion
    // MARK: - Public properties

    var childCoordinators: [BaseCoordinator] = []
```
помоему Private, но проверь